### ISSUE TITLE
Improve the documentation

### ISSUE THREAD

**Issue by jordenchang55** [okgrow/react-native-copilot#45]
There are 5 parameters in the `copilot()` HOC function, but only `overlay` and `animated` are documented.
Can you explain how to use other parameters?

I use this library on Android device, and I follow the example to create the tour guide.
But I found if I don't add `androidStatusBarVisible: false` into `copliot()`, the Modal will have some offset so that part of view will be cut.
In fact, the statusBar is visible on my screen, I don't understand why I should pass `androidStatusBarVisible` as false.

**Comment by RichardLitt** [comment]
The other parameters are listed in the code [here](https://github.com/okgrow/react-native-copilot/blob/master/src/hocs/copilot.js#L29-L35). Do you mean documented in the README, @jordenchang55? If so, yes - we should add a brief note about the other params. Good catch.

**Comment by jordenchang55** [comment]
Yes, it would be better if there is a brief note in README.
IMHO, an user should be able to understand the usage of library by reading README rather than code.

**Comment by mohebifar** [comment]
Good point! I'll need to double check on `androidStatusBarVisible`. PRs are welcome anyways.

**Comment by ohtangza** [comment]
@jordenchang55 I am also experiencing that it shows a wrong overlay with `androidStatusBarVisible : true`. It shows correct one `androidStatusBarVisible: false`. My Android status bar is visible.

**Comment by jordenchang55** [comment]
Not really.
Just like you said, you can get the correct one by setting `androidStatusBarVisible`  as `false`.
I'm not sure which is the right answer.

**Comment by ohtangza** [comment]
@jordenchang55 Thanks, I took your way as well. I'll report if any further information is captured from our user.
